ee/maintained-apps/inputs/homebrew/google-chrome.json

```diff
@@ -4,5 +4,6 @@
   "unique_identifier": "com.google.Chrome",
   "token": "google-chrome",
   "installer_format": "dmg",
```

Comment:
@iansltx can we change this to pkg?

Reply:
Yep. AFK now but can when I'm back at keyboard.